src/ontology/mondo-edit.obo

```diff
@@ -434968,6 +434968,15 @@ intersection_of: MONDO:0005298 ! osteoporosis
 intersection_of: MONDO:0024575 ! pregnancy disorder
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100195
+name: X-linked intellectual disability with hypopituitarism
+def: "An X-linked intellectual disability in which the cause of the disease is a mutation in the SOX3 gene, with variable phenotypes including growth hormone deficiency due to hypopituitarism. It is undetermined if SOX3 is the only gene associated with this disease." [https://clinicalgenome.org/affiliation/40006/, PMID:24450934]
+is_a: MONDO:0020119 {source="https://clinicalgenome.org/affiliation/40006/", source="https://github.com/monarch-initiative/mondo/issues/2045"} ! X-linked syndromic intellectual disability
+intersection_of: MONDO:0020119 {source="https://clinicalgenome.org/affiliation/40006/", source="https://github.com/monarch-initiative/mondo/issues/2045"} ! X-linked syndromic intellectual disability
```

Comment:
nbd, but we generally don't put the he attribution on the equivalence axiom unless warranted. For the MOI you can also relax the axiom and add a subClassOf someValuesFrom and attribute that